parser_library/src/diagnostic.cpp

```diff
@@ -2424,6 +2424,17 @@ diagnostic_op diagnostic_op::mnote_diagnostic(unsigned level, std::string_view m
     return diagnostic_op(lvl, "MNOTE", std::string(message), range, tag);
 }
 
+diagnostic_s diagnostic_s::error_W0001(std::string_view file_name)
+{
+    return diagnostic_s(std::string(file_name),
```

Comment:
I don't remember exactly how these "global" messages are handled, but won't these end up stacking up in the problem panel for ever?

Reply:
I guess this is probably a reason why there was a `TODO` in a first place

Reply:
For whatever reason I couldn't force the error to appear even if I moved it `did_open()` function. So, I converted it back to `todo` with some brief problem description